verilog/tools/kythe/indexing_facts_tree_extractor_test.cc

```diff
@@ -65,14 +66,19 @@ TEST(EqualOperatorTest, FactsTreeExtractor) {
   EXPECT_NE(result_pair4.right, nullptr) << *result_pair4.right;
 }
 
-TEST(ExtractOneFileTest, FactsTreeExtractor) {
+TEST(EmptyCSTTest, FactsTreeExtractor) {
   constexpr absl::string_view code_text = "";
   constexpr absl::string_view file_name = "verilog.v";
   int exit_status = 0;
   bool parse_ok = false;
 
-  const auto expected =
-      T({{Anchor(file_name, 0, code_text.size())}, IndexingFactType ::kFile});
+  const auto expected = T({
```

Comment:
Rewrite top level as a construction: `const T expected( ... );`
(I should do this better everywhere.)